.gitlab-ci.yml

```diff
@@ -84,7 +83,7 @@ stable-test:
   before_script:
     - mkdir -p ./artifacts/sccache/
   script:
-    - cargo +stable build --verbose --all-features
-    - RUST_BACKTRACE=1 cargo +stable test --workspace --verbose --no-default-features
-    - RUST_BACKTRACE=1 cargo +stable test --workspace --verbose --all-features
+    - cargo +stable build --verbose
```

Comment:
why is this needed?

Reply:
Because the integration tests and some unit tests require the compiled debug binary of sccache.